### PR TITLE
Sc query response refactor

### PR DESCRIPTION
**Pull request by sasurobert**
sc query response refactor

### CODE REVIEW

**Review by iulianpascalau — APPROVED** [review]

File: process/smartContract/scQueryService.go [hunk 1]
```diff
@@ -97,11 +97,6 @@ func (service *SCQueryService) executeScCall(query *process.SCQuery, gasPrice ui
 		}
 	}
 
-	err = service.checkVMOutput(vmOutput)
```
Comment (bogdan-rosianu) [inline process/smartContract/scQueryService.go:100]: i think that `checkVMOutput` function remains unsued

File: process/smartContract/scQueryService_test.go [hunk 2]
```diff
@@ -208,10 +208,9 @@ func TestExecuteQuery_WhenNotOkCodeShouldErr(t *testing.T) {
 
 	returnedData, err := target.ExecuteQuery(&query)
 
-	assert.NotNil(t, err)
-	assert.Contains(t, err.Error(), "error running vm func")
-	assert.Contains(t, err.Error(), "add more gas")
-	assert.Nil(t, returnedData)
+	assert.Nil(t, err)
```
Comment (bogdan-rosianu) [inline process/smartContract/scQueryService_test.go:211]: might rename the test(s) because they don't return errors any longer

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: process/smartContract/scQueryService.go (hunk 1 above)
Comment (sasurobert) [reply process/smartContract/scQueryService.go:100]: done

**Review by bogdan-rosianu — DISMISSED** [review]

File: process/smartContract/scQueryService_test.go (hunk 2 above)
Comment (sasurobert) [reply process/smartContract/scQueryService_test.go:211]: done

**Review by iulianpascalau — DISMISSED** [review]

**Review by stefanandy — DISMISSED** [review]
System Test Passed

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by stefanandy — DISMISSED** [review]

**Review by iulianpascalau — DISMISSED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by stefanandy — APPROVED** [review]

**Pull request merged**